src/Mod/Part/App/FT2FC.cpp

```diff
@@ -155,15 +155,15 @@ PyObject* FT2FC(const Py_UNICODE *PyUString,
 //  This sets size to 48 point. Magic.
     error = FT_Set_Char_Size(FTFont,
                              0,             /* char_width in 1/64th of points */
-                             48*64,         /* char_height in 1/64th of points */
+                             480*64,         /* char_height in 1/64th of points */
```

Comment:
Please adjust whitespace uniformity, thanks

Reply:
whitespace is now more consistent